### DISCUSSION TITLE
Data portal wcf to balzor

### DISCUSSION THREAD

**Discussion by mohammedsweif** [MarimerLLC/csla#2723] [Questions]
Hi i have problem and need help now i use csla in wpf app using wcf now i need to make an other app blazor client app i cannot find article show me how can i do that

**Comment by rockfordlhotka** [comment]
Microsoft is no longer supporting WCF on new platforms like Blazor.

I recommend that you switch to using the HttpProxy and an http endpoint.

There is an [open source WCF project](https://github.com/CoreWCF/CoreWCF) to recreate WCF for modern .NET. I haven't followed this project closely, but if you find it stable enough, you can create a data portal channel (or just a client-side proxy?) using this project.
